qa/vagrant/src/test/resources/packaging/tests/example/scripts/is_guide.painless

```diff
@@ -1 +1,5 @@
-_source['title'] == 'Elasticsearch - The Definitive Guide'
+def min = 300;
```

Comment:
I swear I tried to use the Elvis operator but I failed lamentably so I gave up after multiple attempts.

I think I tried something like `int min = params.min_num_pages ?: 300` and got some error message.